spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/StructAccessor.java

```diff
@@ -69,7 +69,11 @@
 					.put(ByteArray.class, AbstractStructReader::getBytes)
 					.put(double[].class, AbstractStructReader::getDoubleArray)
 					.put(long[].class, AbstractStructReader::getLongArray)
-					.put(boolean[].class, AbstractStructReader::getBooleanArray).build();
+					.put(boolean[].class, AbstractStructReader::getBooleanArray)
+					// Note that Struct.class appears in this map. While we support
+					// converting structs into POJO fields of POJOs, the value in this map is for
+					// the case where the field within the POJO is Struct.
```

Comment:
Do we also support the case where the type of the field in the entity is a `Value`?

Reply:
We don't support `Value` here because all columns from the result set come wrapped in `Value`, and what we do with all the convert classes is to get the value from within. 

I suppose somebody could want us to do absolutely no converting, though this would short-circuit a lot of our code, and could be a somewhat large job. I can make an issue of it so we don't lose track of that.

Reply:
#822

Reply:
Thanks!

Reply:
Struct does not have a way to read out just `Value`, so the conversion to the underlying type always needs to be done. Please see the linked issue for more details.